managers/opportunity/index.js

```diff
@@ -54,6 +54,17 @@ class opportunityManager {
         queryObject['opportunityType'] = queryObject.type;
         delete queryObject.type;
       }
+      if (queryObject.female) {
+        queryObject['onlyForFemale'] = queryObject.female == 'true';
```

Comment:
should not this be a boolean value? and please use `===` instead of `==`.

Reply:
Done!